### DISCUSSION TITLE
Option to sort by a random/shuffle order

### DISCUSSION THREAD

**Discussion by ghost** [deckerst/aves#1275] [Ideas]
Hello.
I believe Aves to be the best gallery app on Android, however its currently lacking this major(for me at least) sort option that would make it perfect in my eyes.

I personally find it the most fun to view images/videos in a randomized order and i dont think im the only person that would be happy to have that option.

I know that there exists a shuffle option inside the Slideshow feature but there are two issues i have with it there.

- i think its alot better when you can select the images video from the grid view you have when you selct an album/tag to be able to add some control to the randomness.
- Videos  just play automatically inside the Slideshow feature you cant pause, seek or change the speed there

I would be very grateful if you could add that sort option, it would make it the perfect gallery app to me.

**Comment by deckerst** [comment]
Related issue:
- #933
